src/devices/tuya.ts

```diff
@@ -4903,6 +4902,57 @@ const definitions: Definition[] = [
             tuya.whitelabel('Marmitek', 'SM0202_1', 'Motion sensor', ['_TYZB01_yr95mpib']),
         ],
     },
+    {
+        fingerprint: tuya.fingerprint('TS0001', ['_TZ3210_dse8ogfy']),
+        model: 'FingerBot',
+        vendor: 'TuYa',
+        description: 'Zigbee fingbot plus',
+        whiteLabel: [
+            tuya.whitelabel('Adaprox', 'TS0001_1', 'Zigbee fingbot plus', ['_TZ3210_dse8ogfy']),
+        ],
+        fromZigbee: [fz.on_off, tuya.fz.datapoints],
+        toZigbee: [tz.on_off,
+            {
+                ...tuya.tz.datapoints,
+                key: tuya.tz.datapoints.key.filter((item) => {
+                    return item !== 'state';
```

Comment:
Since tz.on_off is listed first this shouldnt be needed (?)

Reply:
fixed